.github/.env

```diff
@@ -0,0 +1 @@
+CURRENT_KUMA_VERSION= 1.2.2
```

Comment:
Version numbers need to be embedded in the components themselves. Let’s avoid tracking version numbers separately. // @utkarsh-pro @DelusionalOptimist

Reply:
Cool :+1: